tslint.json

```diff
@@ -77,6 +77,18 @@
       "check-format",
       "allow-leading-underscore",
       "allow-pascal-case"
+    ],
+    "whitespace": [true,
+        "check-branch",
+        "check-decl",
+        "check-operator",
+        // "check-module", <- this rule is not compatible with
```

Comment:
Well, it should be. If you have:
```ts
import {
  a,
  b,
} from "foo";
```
I would expect the newline between `b,` and `}` to count as whitespace.